Tigran Kirakosov/flexbox_version/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Dosis:200,300,400,600&display=swap"
+      rel="stylesheet"
+    />
+    <title>Tokyo Weather Widget</title>
+  </head>
+  <body>
+    <li class="container">
+      <section class="widget">
+        <div class="widget__left-side">
+          <div class="widget__main-icon-box">
+            <svg class="widget__main-icon">
+              <use xlink:href="assets/images/sprite.svg#icon-sun" />
+            </svg>
+          </div>
+          <div class="widget__weather-desc">
+            <span>Sunny</span>
+          </div>
+          <div class="widget__weather-date">
+            <span>Tokyo - Wednesday, July 31</span>
+          </div>
+        </div>
+        <li class="widget__right-side">
+          <div class="widget__main-temperature">
+            <span>28 &deg;c</span>
+          </div>
+          <div class="widget__graph-box">
+            <picture class="widget__graph-img">
+              <img src="assets/images/graph.png" alt="weather graph">
+            </picture>
+          </div>
+          <ul class="widget__weekly-weather">
+            <li class="widget__week-day">
+              <div class="widget__icon-box">
+                <svg class="widget__week-icon">
+                  <use xlink:href="assets/images/sprite.svg#icon-sun" />
+                </svg>
+              </div>
+              <ul class="widget__week-temper">
+                <li>Thu</li>
+                <li>28&deg;C</li>
+                <li>18&deg;C</li>
+              </ul>
+            </li>
+            <li class="widget__week-day">
+              <div class="widget__icon-box">
+                <svg class="widget__week-icon">
+                  <use xlink:href="assets/images/sprite.svg#icon-cloudy" />
+                </svg>
+              </div>
+              <ul class="widget__week-temper">
+                <li>Fri</li>
+                <li>22&deg;C</li>
+                <li>20&deg;C</li>
+              </ul>
+            </li>
+            <li class="widget__week-day">
+              <div class="widget__icon-box">
+                <svg class="widget__week-icon">
+                  <use xlink:href="assets/images/sprite.svg#icon-cloudy" />
+                </svg>
+              </div>
+              <ul class="widget__week-temper">
+                <li>Sat</li>
+                <li>30&deg;C</li>
+                <li>27&deg;C</li>
+              </ul>
+            </li>
+          </div>
+        </div>
+      </section>
+    </div>

```

Comment:
Secondly - in the start of file I see `<body><li><section>...`, but in the end - `</section></div></body>`